<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SessionForm</class>
 <widget class="QWidget" name="SessionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>会话信息</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">

   <!-- Top Spacer -->
   <item>
    <spacer name="topSpacer">
     <property name="orientation"><enum>Qt::Orientation::Vertical</enum></property>
     <property name="sizeHint" stdset="0">
      <size><width>20</width><height>20</height></size>
     </property>
    </spacer>
   </item>

   <!-- Centered No Session Label -->
   <item>
    <spacer>
     <property name="orientation"><enum>Qt::Orientation::Vertical</enum></property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="noSessionLabel">
     <property name="text"><string>当前没有会诊记录</string></property>
     <property name="alignment"><set>Qt::AlignCenter</set></property>
     <property name="visible"><bool>false</bool></property>
     <property name="styleSheet"><string notr="true">QLabel { font-size:16px; color:#6b7280; }</string></property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation"><enum>Qt::Orientation::Vertical</enum></property>
    </spacer>
   </item>

   <!-- Cards Container -->
   <item>
    <widget class="QWidget" name="cardsWidget">
     <layout class="QVBoxLayout" name="cardsLayout">
     <!-- Session Card -->
     <item>
      <widget class="QFrame" name="sessionCard">
       <property name="styleSheet">
        <string notr="true">
         QFrame { background: #ffffff; border-radius: 16px; padding: 16px; }
         QLabel { font-size: 14px; color:#374151; background: transparent; margin: 0px; padding: 0px; }
         QLabel#sessionHeader { font-size: 16px; font-weight:bold; margin-bottom:12px; }
        </string>
       </property>
       <layout class="QVBoxLayout" name="sessionLayout">
        <item>
         <widget class="QLabel" name="sessionHeader">
          <property name="text"><string>会诊信息</string></property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="sessionFormLayout">
          <property name="fieldGrowthPolicy"><enum>QFormLayout::ExpandingFieldsGrow</enum></property>
          <property name="labelAlignment"><set>Qt::AlignLeft|Qt::AlignVCenter</set></property>
          <property name="formAlignment"><set>Qt::AlignLeft|Qt::AlignTop</set></property>

          <item row="0" column="0">
           <widget class="QLabel" name="labelSessionTime">
            <property name="text"><string>时间：</string></property>
            <property name="alignment"><set>Qt::AlignLeft|Qt::AlignVCenter</set></property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="sessionTimeEdit">
            <property name="text"><string></string></property>
            <property name="alignment"><set>Qt::AlignLeft|Qt::AlignVCenter</set></property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>

     <!-- Doctor Card -->
     <item>
      <widget class="QFrame" name="doctorCard">
       <property name="styleSheet">
        <string notr="true">
         QFrame { background:#ffffff; border-radius:16px; padding:16px; }
         QLabel { font-size:14px; color:#374151; background:transparent; margin:0px; padding:0px; }
         QLabel#doctorHeader { font-size:16px; font-weight:bold; margin-bottom:12px; }
        </string>
       </property>
       <layout class="QVBoxLayout" name="doctorLayout">
        <item>
         <widget class="QLabel" name="doctorHeader">
          <property name="text"><string>医生信息</string></property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="doctorFormLayout">
          <property name="fieldGrowthPolicy"><enum>QFormLayout::ExpandingFieldsGrow</enum></property>
          <property name="labelAlignment"><set>Qt::AlignLeft|Qt::AlignVCenter</set></property>
          <property name="formAlignment"><set>Qt::AlignLeft|Qt::AlignTop</set></property>

          <item row="0" column="0">
           <widget class="QLabel" name="labelDoctorName">
            <property name="text"><string>姓名：</string></property>
            <property name="alignment"><set>Qt::AlignLeft|Qt::AlignVCenter</set></property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="doctorNameEdit">
            <property name="text"><string></string></property>
            <property name="alignment"><set>Qt::AlignLeft|Qt::AlignVCenter</set></property>
           </widget>
          </item>

          <item row="1" column="0">
           <widget class="QLabel" name="labelDoctorDepartment">
            <property name="text"><string>科室：</string></property>
            <property name="alignment"><set>Qt::AlignLeft|Qt::AlignVCenter</set></property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="doctorDepartmentEdit">
            <property name="text"><string></string></property>
            <property name="alignment"><set>Qt::AlignLeft|Qt::AlignVCenter</set></property>
           </widget>
          </item>

          <item row="2" column="0">
           <widget class="QLabel" name="labelDoctorTitle">
            <property name="text"><string>职称：</string></property>
            <property name="alignment"><set>Qt::AlignLeft|Qt::AlignVCenter</set></property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="doctorTitleEdit">
            <property name="text"><string></string></property>
            <property name="alignment"><set>Qt::AlignLeft|Qt::AlignVCenter</set></property>
           </widget>
          </item>

          <item row="3" column="0">
           <widget class="QLabel" name="labelDoctorDescription">
            <property name="text"><string>简介：</string></property>
            <property name="alignment"><set>Qt::AlignLeft|Qt::AlignVCenter</set></property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="doctorDescriptionEdit">
            <property name="text"><string></string></property>
            <property name="alignment"><set>Qt::AlignLeft|Qt::AlignVCenter</set></property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="messageButton">
          <property name="styleSheet">
           <string notr="true">
            QPushButton {
              background: #d1d5db;
              color: #111827;
              border-radius: 8px;
              padding: 10px;
              font-weight: bold;
            }
            QPushButton:hover:!disabled { background-color: #9ca3af; }
            QPushButton:disabled { background-color: #e5e7eb; color: #6b7280; }
           </string>
          </property>
          <property name="text"><string>发送消息</string></property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     </layout>
    </widget>
   </item>

   <!-- Bottom Spacer -->
   <item>
    <spacer name="bottomSpacer">
     <property name="orientation"><enum>Qt::Orientation::Vertical</enum></property>
     <property name="sizeHint" stdset="0"><size><width>20</width><height>40</height></size></property>
    </spacer>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
